### DISCUSSION TITLE
troika

### DISCUSSION THREAD

**Discussion by giscus** [listenzz/RNDemo#9] [Announcements]
# troika

react native nested scroll，React Native ScrollView 嵌套，react native pull to refresh，React Native 自定义下拉刷新，react native bootom sheet，React Native 底部弹窗

https://todoit.tech/troika.html

**Comment by clin211** [comment]
这个组件支持 content 纵向滚动，tabbar 与 content 进行联动吗？举个例子（伪代码）：
<Content>
        <Tabbar />
        <A />
        <B />
        <C />
        <D />
</Content>

当当前可视区域是B的时候激活 Tabbar 的 B 的 title， 其他的 title 是默认状态，是 C 的时候激活 Tabbar 的 C 的 title

**Reply by listenzz** [reply]
这个组件 NestedScrollView 不包含 Tabbar，需要自己实现。可参考 demo 或引入其它组件库来实现。
